xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -1144,7 +1148,7 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element ref="PrivateCode" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Private code of the vehicle.</xsd:documentation>
+					<xsd:documentation>Private code of the vehicle. DEPRECATED - use privateCodes. -v2.0</xsd:documentation>
```

Comment:
I have the impression that the deprecation had been intended once, but then somehow disappeared from the code. I'm putting it back - OK?

Reply:
Well spotted!